### DISCUSSION TITLE
寻求固定的以阅读模式打开Obsidian指定笔记的方法

### DISCUSSION THREAD

**Discussion by guxianbang** [obsidianzh/forum#55] [Q&A常见问题]
Obsidian目前只支持在阅读/编辑模式之间切换，日常使用中经常在不同笔记、不同状态中来回游走，有时候希望单纯阅读某个笔记，但往往打开时是编辑状态下的笔记，看起来没有阅读模式下简洁美观。

我知道可以设置默认打开方式，但这种方式实测不行，比如当前在编辑模式，你再打开别的笔记依然会是编辑模式。我希望一种不需要肉眼判断当前模式的方式打开。

目前想到的方法是通过“管理工作区布局”来曲线救国，但这种方式的缺点是会覆盖当前的布局。

请问有没有固定的以阅读模式打开（或者固定的以编辑模式打开）指定笔记的方法呢？非常感谢！

**Comment by makerjackie** [comment]
ctrl + E 切换到 阅读模式之后，在阅读模式中点击跳转到其他文件，也默认是阅读模式的吧
